src/allotropy/allotrope/models/shared/definitions/definitions.py

```diff
@@ -30,6 +30,8 @@
 
 TClass = str
 
+TIntValue = int
```

Comment:
Is this used? I couldn't find a reference.

Reply:
It was used when I rolled my own model, before generating one from the embed schema. It could be removed, but it's a valid type and might be used at some point in the future.

Reply:
I think with the re-structuring to use references this would be used again for `repetition setting`. Just need to add it with ref to `tIntValue` and then add it to `definitions.json`

Reply:
Added those changes for repetition setting.